### PR TITLE
docs: update submitting pull request criteria

### PR DESCRIPTION
**Pull request by DipperTheDan**
### Proposed behaviour
Removal of the two bullet points that refer to `gulp test --build` and `.yml`

### Current behaviour
The contribution guidelines refer to `gulp test --build` and `.yml`. 

### Checklist

<del>- [ ] Release notes (Conventional Commits) </del>
<del>- [ ] Unit tests</del>
<del>- [ ] Cypress automation tests</del>
<del>- [ ] Storybook added or updated</del>
<del>- [ ] Theme support</del>
<del>- [ ] Typescript `d.ts` file added or updated</del>

### Additional context
N/A

### Testing instructions
N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,11 +53,9 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
 
 When submitting a pull request, please ensure your branch meets the following criteria:
 
-* It is a passing build (`gulp test --build` returns success).
 * The code is fully tested and has full coverage.
 * The code follows our coding guidelines (TBC).
 * The code does not introduce unnecessary dependencies (no jQuery!).
-* Ensure descriptive `.yml` file is added to the `./change/next` directory.
```
Comment (chrisbarber86) [inline CONTRIBUTING.md]: Can we document in this file the commit strategy we are using and how that will ultimately form part of the release notes

https://github.com/Sage/carbon/blob/21b66cb43312635f58134844fad5281de03da8ab/rfcs/text/semantic-release.md#release-notes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,11 +53,9 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
 
 When submitting a pull request, please ensure your branch meets the following criteria:
 
-* It is a passing build (`gulp test --build` returns success).
```
Comment (chrisbarber86) [inline CONTRIBUTING.md]: Can we direct people to somewhere in the README that explains how to run the tests?

Comment (DipperTheDan) [reply CONTRIBUTING.md]: Good idea. I'll get that added to this file.

File: CONTRIBUTING.md (hunk 1 above)
Comment (DipperTheDan) [reply CONTRIBUTING.md]: I'll get that added in.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,11 +53,10 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
...
+* It is a passing build. https://github.com/Sage/carbon/blob/master/README.md#testing
```
Comment (jamime) [inline CONTRIBUTING.md]: `https://github.com/Sage/carbon/blob/master/README.md#testing` doesn't explain how to run the tests.

**Comment by DipperTheDan** [comment]
No longer required as it duplicates work already being carried out by the Front End Squad. PR is being closed.

**Pull request closed without merging**